spec/unit/new-spec.ts

```diff
@@ -1,13 +1,31 @@
+import * as fs from "fs-extra";
+import { parse } from "path";
 import { default as newCmd } from "../../lib/commands/new";
 import { ProjectConfig } from "../../lib/ProjectConfig";
 import { PromptSession } from "../../lib/PromptSession";
 import { Util } from "../../lib/Util";
 import { resetSpy } from "../helpers/utils";
 
 describe("Unit - New command", () => {
+	let testFolder = parse(__filename).name;
```

Comment:
Ummm are you sure you need actual folders? Unit tests don't have that setup for a reason - they don't actually write any files at all. There's already a spy on `generateFiles` as far as I see.

Reply:
It's not needed. This was written using the CPDD.

Reply:
😆